src/handlers/store-standardised-invoices/field-utils/get-highest-confidence-textract-value.test.ts

```diff
@@ -0,0 +1,102 @@
+import { Textract } from "aws-sdk";
+import { getHighestConfidenceTextractValue } from "./get-highest-confidence-textract-value";
+
+const mockField = (
+  type?: string,
+  value?: string,
+  confidence?: number
+): Textract.ExpenseField => ({
+  Type: {
+    Text: type,
+    Confidence:
+      confidence === undefined ? undefined : Math.sqrt(confidence / 100) * 100,
+  },
+  ValueDetection: {
+    Text: value,
+    Confidence:
+      confidence === undefined ? undefined : Math.sqrt(confidence / 100) * 100,
+  },
+});
+
+describe("Highest confidence Textract value getter", () => {
+  let givenType: string;
+
+  beforeEach(() => {
+    givenType = "given type";
+  });
+
+  test("Highest confidence Textract value getter with no fields", () => {
+    const givenFields: Textract.ExpenseField[] = [];
```

Comment:
you could have that as a variable with the type at the beginning to save the repetition of the `const` and type.